### ISSUE TITLE
Feature Request: Repeatable Field  / Group ?

### ISSUE THREAD

**Issue by coolcat-creations** [joomla-projects/custom-fields#96]
Is it possible to set up fields or fieldgroups repeatable? For example in a recipe i want to have the amount as one textfield and the ingredientname as another field. 
The userinput should look like this then:
![amountname](https://cloud.githubusercontent.com/assets/828371/15711142/23771c36-280d-11e6-821b-6371bbd2e19e.jpg)

**Comment by laoneo** [comment]
I guess this can't make it into the 3.7 as I just don't have time. If somebody would volunteer on that, I'm happy to merge the PR.
I'm not sure, but will the repeatable option not being replaced by a new feature which allows to use a form in a field https://github.com/joomla/joomla-cms/pull/7829.

**Comment by wilsonge** [comment]
Yeah subforms are going to replace repeatable. subforms are going to be complex to replicate though I think. Definitely wanted - but probably not essential for 3.7 - very nice if we can have it though

**Comment by webchun** [comment]
+1
Repeateable Field or (maybe) Subforms will be useful when we want to create a custom function like image slideshow in the content
